### DISCUSSION TITLE
Clone with original dates

### DISCUSSION THREAD

**Discussion by diegolaz79** [community/community#31374] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, is there a way to clone a repositories with the files original dates? 
In our team we use a lot of file/dll dates to verify versions... but each time you clone a repo, all files get the current date locally....is there a way to clone, but keep the date from the repository?

Thanks!

**Comment by airtower-luna** [comment, marked as answer]
There isn't any such thing.

> In our team we use a lot of file/dll dates to verify versions

That sounds like a problematic approach for multiple reasons:

* Keeping DLLs or other binaries in Git (or SCMs in general) is usually a bad approach. The repository should contain the sources to build them, if you want to distribute pre-built binaries there are usually better ways (depending on the ecosystem you move in).
* The date doesn't contain any information about API, ABI, or anything else. It's not version information.

With Make and similar tools using the file timestamp purely locally to estimate which files are current and which need to be rebuilt works reasonably well in most cases, but even then it's well-advised to test a clean build now and then.

**Comment by diegolaz79** [comment]
Thank you! And I agree, needed to confirm there is no option for that.
